packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -2454,6 +2454,50 @@ void main() {
 
     expect(menuAnchor.alignmentOffset, alignmentOffset);
   });
+
+  testWidgets('DropdownMenu filter is disabled until text input', (WidgetTester tester) async{
+    await tester.pumpWidget(MaterialApp(
+      home: Scaffold(
+        body: DropdownMenu<TestMenu>(
+          requestFocusOnTap: true,
+          enableFilter: true,
+          initialSelection: menuChildren[0].value,
+          dropdownMenuEntries: menuChildren,
+        ),
+      ),
+    ));
+
+    await tester.tap(find.byType(DropdownMenu<TestMenu>));
+    await tester.pumpAndSettle();
+    // All entries should be avaliable, and two buttons should be found for each entry.
```

Comment:
```suggestion
    // All entries should be available, and two buttons should be found for each entry.
```

Small typo, and maybe add an empty line before this comment